com.tiny-beast-dictionary.yml

```diff
@@ -0,0 +1,29 @@
+app-id: com.dict.Tiny-beast-dictionary
```

Comment:
you own `dict.com` ?

Reply:
I'm one of its shareholders.
Do you expect me to read all those vague docs just for an ID?
Is it supposed to be io.github.tbdict? Io.github.tbdict.tbdict? Or maybe io.github.mygithubusername.tbdict?

BTW, snapcraft docs are clearer than flatpak. I don't know why many use the latter.

Reply:
While I understand the frustration, patches welcome. Nobody forces you to submit your app to Flathub either.

Being a shareholder doesn't imply an ownership. It should be io.github.Arian94.Tiny-beast-dictionary, as you could have figured it out by checking repository list at https://github.com/flathub/.